mautic/mautic
# Fix: A border on the mj-column is shown twice in the editor

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [n]
| Automated tests included?              | [ n]
| Related user documentation PR URL      | 
| Related developer documentation PR URL |
| Issue(s) addressed                     | Fixes #11207

#### Description:
See https://github.com/mautic/mautic/issues/11207

In addition I updated all JS dependencies. Mainly grapesjs itself:
from  "grapesjs": "^0.17.29" to "grapesjs": "^0.18.4"

There were some breaking changes, but as far as I can tell Mautic is not affected by them:
https://github.com/artf/grapesjs/releases

#### Steps to test this PR:
1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Add a border to a section: `<mj-column border="4px solid #F45E43">`
3. It should be displayed only one border. Not like this: 
<img width="681" alt="image" src="https://user-images.githubusercontent.com/13075514/171410726-7a48d3a7-4cef-4329-9fca-8bc849daac34.png">
4. All other email builder functionality should work as expected 

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11209"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
